poshc2/server/Implant.py

```diff
@@ -121,9 +121,8 @@ def autoruns(self):
         if "C#" in self.Pivot:
             new_task("loadmodule Stage2-Core.exe", "autoruns", self.RandomURI)
             new_task("loadmodule PwrStatusTracker.dll", "autoruns", self.RandomURI)
-            update_mods("Stage2-Core.exe", self.RandomURI)
-            update_mods("PwrStatusTracker.dll", self.RandomURI)
             new_task("loadpowerstatus", "autoruns", self.RandomURI)
+            update_mods("Stage2-Core.exe PwrStatusTracker.dll", self.RandomURI)         
```

Comment:
This would maybe be better being stored as a JSON list or similar and being parsed as a list? Then can search it, append to it, remove from it etc.

`["Stage2-Core.exe", "PwrStatusTracker.dll"]` etc.

Can maybe change the way it's done in the future